torch/csrc/stable/c/shim.h

```diff
@@ -37,6 +37,34 @@ AOTI_TORCH_EXPORT AOTITorchError torch_library_impl(
     void (*fn)(StableIValue*, uint64_t, uint64_t),
     uint64_t extension_build_version);
 
+struct StableListOpaque;
+using StableListHandle = StableListOpaque*;
+
+// returns an owning reference of a StableList. callee is responsible for
+// freeing memory.
+AOTI_TORCH_EXPORT AOTITorchError
+torch_new_list_reserve_size(size_t size, StableListHandle* ret);
+
+// new list handle steals ownership of underlying elements from orig_handle
+AOTI_TORCH_EXPORT AOTITorchError torch_new_list_handle(
+    StableListHandle orig_handle,
+    StableListHandle* new_handle);
+
+AOTI_TORCH_EXPORT AOTITorchError
+torch_list_size(StableListHandle list_handle, size_t* size);
+
+AOTI_TORCH_EXPORT AOTITorchError torch_list_get(
+    StableListHandle list_handle,
+    size_t index,
+    StableIValue* element);
```

Comment:
torch_list_get_item?

Reply:
But this one should be `torch_list_at(...)`